_posts/2023-04-09-haberdashers-win-sr2023.md

```diff
@@ -0,0 +1,110 @@
+
+To help teams locate the tokens there were 2D barcode style markers attached that our computer vision library could detect. However each of the markers identified themselves as exactly the same, so teams had to use other sensors to detect which type of token is which. Bronze and Silver tokens were the same size but the Silver tokens weighed 300g more, and Gold tokens were larger than both Bronze and Silver. Teams were scored at the end of the match for collecting the most points.
+
+{% include figure.html class="right" src="/images/content/blog/sr2023/arena.jpg"
+           caption="The 'Greed' arena" %}
+
+## Virtual Competition
+
+The first part of our league this year was a virtual competition, broadcast live from Southampton in March. Across twenty matches, teams competed in a simulated version of the main competition, testing their coding skills against one another. The day was handily won by Haberdashers’ College, coming 1st in all four of their matches, and going on to win the competition as a whole.
+
+## League Matches
+
+At the in person event, teams participated in a further 70 league matches, showcasing a diverse range of robot designs and strategies. The robots displayed impressive agility, precision, and ingenuity, as they navigated the arena to capture tokens and outmanoeuvre their opponents. Team’s had varying designs from suction systems, lifting mechanisms, and even a flywheel designed to launch tokens backwards.
+
+## Knockouts
+
+The knockout rounds saw teams from the league matches pitted against one another in intense, high-stakes battles. With each round, the competition grew fiercer, and the robots were pushed to their limits as they raced to collect tokens and secure their place in the finals.
+
+## The Final
+
+{% include figure.html class="left" src="/images/content/blog/sr2023/final.JPG"
+           caption="Beginning of the SR2023 Final" alt="Three competitors placing their robots into the arena with a crowd of onlookers watching" %}
+
+In the nail-biting final, Haberdashers’ School faced off against Abingdon School, Gymnasium Markt Indersdorf, and Harris Westminster Sixth Form. Harris Westminster Sixth Form managed to successfully grab Haberdashers’ gold token, but were not able to get it solely back to their own scoring zone. Whilst this was happening, Gymnasium Markt Indersdorf tried to grab a gold token with their arms, but missed slightly and instead grabbed the Abingdon School robot, slowly pulling on the robot and some of its wires for the rest of the match! At the same time, the Haberdashers’ robot slowly but steadily collected bronze and silver tokens and deposited them in their zone, before going back and getting more. The teams displayed exceptional skill and teamwork, and after a fierce battle, Haberdashers’ School emerged victorious, claiming the first-place title in Student Robotics 2023.
+
+## Results
```

Comment:
Is there anything we can do to force this down a line? Currently on the "blog" pages (rather than the individual post pages) the flow looks like this:
![image](https://user-images.githubusercontent.com/336212/230959648-6a08717b-eb19-428a-b93c-42edfdebe01d.png)

Reply:
Can't replicate this on Windows using Chrome or Firefox. What browser are you using? 
![image](https://user-images.githubusercontent.com/1496834/231560228-eafdd160-2720-482d-895f-e0e42f797c49.png)

Reply:
Firefox on Linux also works well

![image](https://user-images.githubusercontent.com/6527489/231571141-5951c605-57b7-492e-968c-50147d861ef5.png)